settings.gradle

```
@@ -23,8 +23,7 @@ def spockBuildCacheUsername = ext.has('spockBuildCacheUsername') ? ext['spockBui
 def spockBuildCachePassword = ext.has('spockBuildCachePassword') ? ext['spockBuildCachePassword'] : null
 
 def accessKeysAreMissing() {
-  File accessKeys = new File(gradle.gradleUserHomeDir, "enterprise/keys.properties")
-  return !System.env['GRADLE_ENTERPRISE_ACCESS_KEY'] && (!accessKeys.file || !accessKeys.text?.contains("ge.spockframework.org"))
+  return !System.env['GRADLE_ENTERPRISE_ACCESS_KEY']?.trim()
```

This might be easier to read:
```suggestion
  return !(System.env['GRADLE_ENTERPRISE_ACCESS_KEY']?.trim())
```